packages/vue-generator/src/generator/vue/sfc/generateTemplate.js

```
@@ -42,6 +42,10 @@ export const handleTinyIcon = (nameObj, globalHooks) => {
     return
   }
 
+  // 增加 svg 颜色属性，使编辑模式与出码模式一致
+  debugger
+  nameObj.schema.props['fill'] = 'currentColor'
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)